cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -1626,3 +1626,26 @@ def _test_video_feature(self, flag, key, override_fn, is_enabled):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json()[key], is_enabled)
+
+
+class GetStorageBucketTestCase(TestCase):
```

Comment:
In case of any connection issue in imports this test fails. 
`cms/djangoapps/contentstore/views/tests/test_videos.py::GetStorageBucketTestCase::test_storage_bucket - NameError: name 'S3Connection' is not defined
`